hugo/content/contribute.md

```diff
@@ -3,83 +3,261 @@ title: Contribute
-## Mumble itself
+#### Other 3rd party projects
+
+If you are interested in more projects (Clients, Servers, cli tools etc.), you can also take a look at {{< wiki 3rd_Party_Applications />}}.
+<!-- https://wiki.mumble.info/wiki/3rd_Party_Applications -->
+Many of these projects will either gladly accept help or can (often) be forked easily, because of the open licenses.
+
+
+## Non-Coding
+
+### Translation
+
+Mumble currently (April 2020) supports over 15 different languages (38 if including unfinished translations). As more people get to know and use Mumble, versions in their native languages become more and more important.
+
+We are thankful for any contribution on translating. Please check the {{< wiki "Language Translation" "Language Translation wiki page" />}} for more information.
+
+### Testing
+
+To find bugs and polish new releases (before and after release) we need testers.
+
+If you want to test the newest development versions, you have two options:
+
+- We provide development snapshots. You can download them from our [Downloads page]({{< relref "downloads" >}}).
+<!-- link to exact point? https://www.mumble.info/downloads/#development-snapshots -->
+
+- Build the latest master of Mumble, see [Building Mumble](https://github.com/mumble-voip/mumble#building).
+
+If you find any issues, take a look at [Report Bugs](#report-bugs-and-feature-requests).
+
+<!-- mention documentation? Potential problem with TOC linking as the categories have the same name -->
+<!-- 
+### Documentation
+
+See [below](#documentation). -->
+
+### Mumble theme
+
+<!-- Add more details, descriptions etc.? -->
+You find the official Mumble theme in its own [repository on GitHub](https://github.com/mumble-voip/mumble-theme).
+
+#### Additional themes
+
+Additional Mumble themes bring variety into Mumble's design and choices for different tastes.
+
+Create new themes for Mumble! Information can be found on our {{< wiki Skinning "Skinning wiki page" />}} and existing skins we know about on the {{< wiki Skins "Skins wiki page" />}}.
+
+<!-- Wiki links should be renamed, once the naming in the wiki has changed. -->
+
+
+# Documentation & Website
+
+Improve our Documentation and Website.
+
+## Documentation
+
+<!-- Todo: 
+
+- We should add guidelines for style etc.
+Maybe we could put that somewhere in the github repo instead.
+
+See also: https://github.com/mumble-voip/mumble-www/discussions/125 and https://github.com/mumble-voip/mumble/discussions/4648
+
+Add links to issues for a concrete plan:
+
+- add Labels for important issues and pull requests
+
+-->
+
+We plan to move all our documentation to the Website and GitHub Repo respectively.
+
+Right now you find most of the Documentation in our
+[wiki](https://wiki.mumble.info).
+
+Any contributions are welcome, improving existing pages or creating new ones with useful content. Also see the {{< wiki "Category:Please_Improve" "Please Improve Category" />}}.
+
+The new places for documentation will probably be here:
```

Comment:
Could we have a decision about the places (seperate documentation repo e.g.) and some content ideas & structure concept before merge?
Otherwise people will not really know what to do and how to help, aside from editing the existing wiki.
This could of course also be an intermediate solution, to tell people to just edit the wiki for now.

I know we already had this discussion, but it's no solution to simply say: there will be no answer.
No answer, no content.

For example: https://github.com/mumble-voip/mumble-www/discussions/125#discussioncomment-274784

Reply:
I will add the discussion on whether we want a separate repo or not to the list of topics for our next meeting :point_up:

Reply:
To summarize additional main questions:
1. Should we encourage potential contributors to create new markdown files for documentation (also by copying & updating the wiki information)?
2. Is it planned to remove the wiki enterely?
3. Is it planned to somehow automatically convert the wiki?
4. Does someone in the team want to make a (short) plan for the transition and new documentation?
So contributors have a rough idea about the structure (Categories, Topics, what should be split), potential and wanted contents, writing style etc. (also in regards to the existing [contents of the wiki](https://wiki.mumble.info/wiki/Special:AllPages)).

Reply:
See https://github.com/mumble-voip/mumble-www/discussions/125#discussioncomment-287246

Reply:
Ok, so I think we now have three options for this page:
1. Only mention the wiki for now.
2. Mention the wiki, but tell users that a change is planned (stay tuned)
3. Option 2 + that people are free to already contribute something for user documentation (in this repo) and/or development documentation (in the main repo).

Reply:
I think we should refer them to he wiki for the time being

Reply:
Ok.